src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$gap-size: 1px;
+$squere-size: 100px;
+$calendar-size: ($gap-size + $squere-size) * 7 ;
+$days-day: 
+  "mon" 0,
+  "tue" 1,
+  "wed" 2,
+  "thu" 3,
+  "fri" 4,
+  "sat" 5,
+  "sun" 6;
```

Comment:
Try to correct $days-day so it will work correct. Like this - if it's tuesday margin-left should be size of cell (100px) + gap (1px), wednesday - margin-left should be 2 * (size of cell + gap) etc.

Reply:
Thank you, can't get it by myself